README.md

```diff
@@ -1,61 +1,78 @@
 # CislunarSim
 
-Simulation Software for the Cislunar Explorers Mission
+Simulation Software for the Cislunar Explorers Mission  
+[Read the Docs](https://cislunarsim.readthedocs.io/en/latest/)
 
 ## Getting Started
 
+Create a virtual Python environment and install all the necessary dependencies. Make sure you run this from the root of the repository!
+
 ```bash
 python -m venv venv
 source venv/bin/activate
 pip install -e .
 ```
 
-# Plotting sim runs
+You can exit this environment by running `deactivate`. To re-enter an existing environment, all you need is `source venv/bin/activate`.  
 
-Input structure:
+*Note: Some problems have been known to occur if your [pip](https://pypi.org/project/pip/) is not updated. Follow the warning prompt your terminal gives if this happens.*
 
-```
-python3 src/utils/plot.py {file path}
-```
+## Running the Sim
 
-Example:
+#### Usage:
 
+```zsh
+python src/main.py {file path} [-v]
 ```
-python3 src/utils/plot.py runs/cislunarsim-1650068219.9772968.csv
+
+#### Options:  
+`file path` *(Required)*: The path of the config file to simulate  
+`-v` *(Optional)*: Verbose mode for logging extra information to the terminal
+
+#### Examples:  
+```zsh
+python src/main.py configs/freefall.json 
+python src/main.py configs/test_angles.json -v
 ```
 
-## Running the sim
+## Plotting Sim Runs
+
+#### Usage:  
 
-## IMPORTANT: Python Version MUST be 3.8 and above
+```zsh
+python src/utils/plot.py {file path}
+```
 
-Run
+#### Options:  
+`file path` *(Required)*: The path of the csv file to plot  
 
-Input structure:
-`"python3 src/main.py {file path} [-v]"`
-File path is a required argument, verbose is an optional argument.
-Example:
-`"python3 src/main.py configs/freefall.json" "python3 src/main.py configs/test_angles.json -v"`
+#### Example:  
+```zsh
+python src/utils/plot.py runs/cislunarsim-1650068219.9772968.csv
```

Comment:
Let's add a note that this example won't work initially for a person because the CSV's are generated locally.